app/views/govuk_publishing_components/component_guide/show.html.erb

```diff
@@ -2,12 +2,10 @@
 <%= render 'govuk_publishing_components/components/title', title: @component_doc.name, context: "Component" %>
 
 <% if @component_doc.part_of_admin_layout? %>
-<div class="component-warning">
-  <h2 class="component-warning__title">Admin layout only</h2>
-  <p>
-    This component uses the Design System, so only works inside the <%= link_to "admin layout", "/component-guide/layout_for_admin" %>
-  </p>
-</div>
+<%= render "govuk_publishing_components/components/notice", {
```

Comment:
Done in https://github.com/alphagov/govuk_publishing_components/pull/411

Reply:
Rebased after #411 was merged.